### ISSUE TITLE
Cannot initiate compton

### ISSUE THREAD

**Issue by ImaginePhoenix** [tryone144/compton#40]
Installed from the AUR package. Having problem to initiate. When attempted to launch from the terminal, it returns
```
$ compton --config ~/.config/compton.conf
glx_hasglxext(): Missing GLX extension GLX_EXT_texture_from_pixmap.
```
config has been given [here](https://pastebin.com/52iGWi8B).

Tried overriding the config file parameters with
```
compton --blur-background --blur-method kawase --blur-strength 5 --glx-no-rebind-pixmap false --glx-use-copysubbuffermesa true
```
Which shows,
```
get_cfg(): compton doesn't accept positional arguments.
```
`glxinfo | grep -i opengl` shows,
```
OpenGL vendor string: X.Org
OpenGL renderer string: AMD HAINAN (DRM 2.50.0, 5.4.30-1-lts, LLVM 9.0.1)
OpenGL core profile version string: 4.5 (Core Profile) Mesa 20.0.4
OpenGL core profile shading language version string: 4.50
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile
OpenGL core profile extensions:
OpenGL version string: 4.5 (Compatibility Profile) Mesa 20.0.4
OpenGL shading language version string: 4.50
OpenGL context flags: (none)
OpenGL profile mask: compatibility profile
OpenGL extensions:
OpenGL ES profile version string: OpenGL ES 3.2 Mesa 20.0.4
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
OpenGL ES profile extensions:
```

**Comment by tryone144** [comment]
Please provide the full output of `glxinfo` (via pastebin or similar). But it looks as if your OpenGL driver is missing the required `GLX_EXT_texture_from_pixmap` extension that is required by the glx backend. Maybe more information on your hardware and OS could help.

The version you are using isn't actively maintained anymore in favor of the active fork https://github.com/yshui/picom. But this isn't the root cause of your issue.

**Comment by ImaginePhoenix** [comment]
OS: Arch Linux
Kernel: 5.4.30-1-lts
I have a hybrid GPU system with Intel HD Graphics 4400 integrated and AMD Radeon HD 8670M dedicated.
Output of `inxi -G` 
```
Graphics:
  Device-1: Intel Haswell-ULT Integrated Graphics driver: i915
  v: kernel
  Device-2: AMD Sun XT [Radeon HD 8670A/8670M/8690M / R5 M330 / M430 /
  Radeon 520 Mobile]
  driver: radeon v: kernel
  Display: x11 server: X.Org 1.20.8 driver: intel,modesetting
  unloaded: vesa resolution: 1366x768~60Hz
  OpenGL: renderer: AMD HAINAN (DRM 2.50.0 5.4.30-1-lts LLVM 9.0.1)
  v: 4.5 Mesa 20.0.4
```
Output of `glxinfo` is [here](https://pastebin.com/MEKfyTpN).

**Comment by tryone144** [comment]
From your `glxinfo` output, your dGPU actually supports the required GLX extension.
I have never worked with those hybrid systems and can't say for sure whats wrong, but maybe there is some confusion with which driver/GPU is used for the X server between iGPU and dGPU. Hopefully the [ArchWiki](https://wiki.archlinux.org/index.php/Hybrid_graphics) has some useful information for further testing.

**Comment by ImaginePhoenix** [comment]
Well, I went through the wiki pages and your man-page as well but could not figure out what may have went wrong. However, I tried copying the config from [Brodie Robertson](https://github.com/BrodieRobertson/dotfiles/blob/master/config/compton.conf) and blur seems to perfectly work now. I am still not sure what parameter I missed or set incorrectly, causing the issue but it's clear it was my mistake that the blur did not work earlier. I am closing the thread now. Thank you for your attention.
